includes/payment-methods/class-wc-stripe-upe-payment-gateway.php

```
@@ -4197,6 +4206,43 @@ public function get_validated_option( string $field_key, $empty_value = null ) {
+
+			// parent::get_tokens() already loaded the main stripe gateway.
+			if ( $this->id === $gateway_id ) {
+				continue;
+			}
+
+			$method_tokens = WC_Payment_Tokens::get_customer_tokens( get_current_user_id(), $gateway_id );
+			$tokens        = array_merge( $tokens, $method_tokens );
+		}
+
+		return array_unique( $tokens );
+	}
```

PHPMD (2.15.0)

[error] The method get_tokens is not named in camelCase. (undefined)

(CamelCaseMethodName)